src/components/Filter.js

```diff
@@ -0,0 +1,30 @@
+import { useRef } from 'react';
+
+const Filter = (props) => {
+
+    // check user input and update filter state to searched term
```

Comment:
I appreciate that you included some notes in your code - I think it really helps guide the reader through your thought process